app/_src/gateway/production/deployment-topologies/kubernetes.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Kubernetes Topologies
+content_type: explanation
+---
+
+Running {{ site.base_gateway }} on Kubernetes is a common deployment pattern. When running on Kubernetes, there are two configuration options available:
+
+* [Hybrid mode](#hybrid-mode), with a CP (either on-prem or via Konnect) and multiple DPs
+* [DB-less mode](/gateway/{{ page.release }}/production/deployment-topologies/db-less-and-declarative-config/), configured using CRDs and the {{ site.kic_product_name }}
+
+Running Kong in Hybrid mode is commonly referred to as "Kong _on_ Kubernetes". Running Kong with {{ site.kic_product_name }} is commonly referred to as "Kong _for_ Kubernetes", as it provides a Kubernetes native way of configuring Kong entities using {{ site.kic_product_name }}.
```

Comment:
I was asked not to use "Kong" in this way to mean KGW

Reply:
Understood. This is historical nomenclature so I'm comfortable keeping it for now